openapi/newsapi/Package.md

```diff
@@ -0,0 +1,17 @@
+Connects to News API from Ballerina
+
+#### Compatibility
+|                               | Version               |
+|-------------------------------|-----------------------|
+| Ballerina Language Version    | **Swan Lake Alpha 5** |
+|       News API Version        |         **V2**        |
+
+### Package Overview
+The <newsapi> is a [Ballerina](https://ballerina.io/) connector for News API.
```

Comment:
why do you have wording like this `<newsapi>`? can't we simply say `newsapi.org`?

Reply:
it is the package name. can i place it as `newsapi` only instead of `newsapi.org`

Reply:
`newsapi` is fine.